CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ set(CMAKE_CXX_STANDARD 17)
 
 # Build options.
 option(USE_DRM "Select WAYLAND or DRM as the display backend type" OFF)
+option(USE_EGLSTREAM "Select WAYLAND or EGLStream as the display backend type" OFF)
```

Comment:
Can you change USE_DRM and USE_EGLSTREAM to `BACKEND_TYPE=Drm-gbm`, `BACKEND_TYPE=Drm-eglstream`, `BACKEND_TYPE=Wayland` etc?

Reply:
I changed it to `BACKEND_TYPE=xxx`.